docs/gettingstarted/index.rst

```diff
@@ -159,6 +159,9 @@ Getting started with Flocker
          .. version-code-block:: console
 
             you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
+              curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt && \
```

Comment:
You need to download the crt and key files from the downloads directory in the docs build for this branch, the link in the cURL command is wrong because there's no release including this branch. But that won't be a problem once it's merged. This is interpolating the `latest-installable` thing.